### ISSUE TITLE
Latest build does not run on 1.9.1

### ISSUE THREAD

**Issue by gfxguru** [dweeves/magmi-git#220]
All types of issues, had to go back to a previous build

**Comment by intrepidws** [comment]
Is this still an issue?

**Comment by RiccardoB** [comment]
Yes there still issues like the one described here https://github.com/dweeves/magmi-git/issues/183 on images import.
